public/model/map-coloring.js

```diff
@@ -58,6 +62,14 @@ var MapColoring = P({
     return layer_name_to_data;
   },
 
+  is_country_selected: function(country_code) {
+    return this.selected_countries.is_country_selected(country_code);
+  },
+
+  get_selected_countries: function() {
```

Comment:
nit. is this proxy function necessary? not really a big deal in the grand scheme of things ...